packages/firestore/test/integration/util/helpers.ts

```diff
@@ -124,6 +126,20 @@ export const apiDescribe = apiDescribeInternal.bind(
 apiDescribe.skip = apiDescribeInternal.bind(null, describe.skip);
 apiDescribe.only = apiDescribeInternal.bind(null, describe.only);
 
+export class MockCredentialsProvider extends EmptyCredentialsProvider {
+
+  private listener: CredentialChangeListener | null = null;
+
+  changeUserTo(user: User): void {
```

Comment:
Do you think it makes sense to call this `handleUserChange` to match SyncEngine?

Reply:
`handleUserChange` implies it's handling something, which is not true. Changed to `triggerUserChange` instead.